databricks/koalas/series.py

```diff
@@ -3529,7 +3529,42 @@ def __len__(self):
         return len(self.to_dataframe())
 
     def __getitem__(self, key):
-        return self._with_new_scol(self._scol.__getitem__(key))
+        if not isinstance(key, (str, tuple)):
```

Comment:
Why should we restrict the type to only strings? From rough reading, it looks going to work. e.g.:

```python
>>> pd.Series([10, 20, 15, 30, 45], [1, 1, 3, 4, 5])[1]
1    10
1    20
```

Reply:
ah. it should not restrict. i just fixed them. Thanks!! 😄